dropwizard-jersey/src/test/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapperTest.java

```diff
@@ -97,7 +97,10 @@ public void returnsA400ForNonDeserializableRequestEntities() throws Exception {
 
     @Test
     public void returnsA400ForWrongInputType() throws Exception {
```

Comment:
Not sure if this helps, but there are couple of other `JsonMappingException` subtypes available, beside ones already checked for (`InvalidFormatException`, `PropertyBindingException`):

* `IgnoredPropertyException`
* `InvalidTypeIdException` (added in 2.8)
* `UnrecognizedPropertyException` (subtype of `PropertyBindingException`)

Also if there is something small I could do wrt exception message I could consider changes for 2.8.2. I hope to keep messages (more) stable going forward. 2.8 changes were to try to unify messages, as well as allow more customized handling of problems via `DeserializerProblemHandler` extensions.